libc/cmake/modules/LLVMLibCArchitectures.cmake

```diff
@@ -171,6 +171,11 @@ elseif(LIBC_TARGET_OS STREQUAL "poky")
   # include the right directories during compilation.
   set(LIBC_TARGET_OS_IS_LINUX TRUE)
   set(LIBC_TARGET_OS "linux")
+elseif(LIBC_TARGET_OS STREQUAL "suse")
```

Comment:
We could probably merge this with the `poky` case above.

Reply:
Updated.